bower.json

```diff
@@ -32,6 +32,7 @@
         "requirejs-plugins": "~1.0.2",
         "At.js": "emord/At.js#override-insert",
         "Caret.js": "0.2.2",
+        "fuse.js": "1.2.2",
```

Comment:
I'm going to keep it at 1.2.2 since it's a new library from someone I'm not familiar with. If we notice that there are new features when a new version is released, it'll probably be good to decide if we can trust it.